src/kili/services/label_data_parsing/bounding_poly.py

```diff
@@ -76,10 +76,6 @@ def normalized_vertices(
         vertices = normalized_vertices if self._type_of_tool else normalized_vertices[0]
         for vertex in vertices:
             assert isinstance(vertex, dict), f"Vertex {vertex} is not a dict."
-            if not 0 <= vertex["x"] <= 1:
```

Comment:
for me, this should stay here add we should add a new field for not normalized vertices